documenation/documentation.html

```diff
@@ -64,6 +64,10 @@
         <div class="nav-item" id="responsive-toast-id">Toast</div>
         <div class="nav-item" id="responsive-list-id">List</div>
         <div class="nav-item" id="responsive-rating-id">Rating</div>
+        <div class="nav-item" id="responsive-modal-id">Modal</div>
+        <div class="nav-item" id="responsive-navigation-id">Navigation</div>
```

Comment:
Here you can use the `li` tag instead of `div`. It's always a good practice to use semantic tags.